components/count-badge/count-badge.js

```diff
@@ -134,7 +160,13 @@ class CountBadge extends RtlMixin(LitElement) {
 			numberString = `${'9'.repeat(this.maxDigits)}+`;
 		}
 		return html`
-        	<div class="d2l-count-badge-number">${numberString}</div>`;
+        	<div
+			tabindex="${ifDefined(this.tabStop ? '0' : undefined)}" 
+			role="${ifDefined(this.announceChanges ? 'status' : undefined)}"
+			aria-label="${this.description}">
+				<div class="d2l-count-badge-number" aria-hidden="true">${numberString}</div>
+				<span class="d2l-offscreen">"${this.description}"</span>
+			</div>`;
```

Comment:
This might be more of a question for @geurts, but what do you think about adding a `<d2l-tooltip>` with the description/text that they're required to provide? It would have the advantage of showing the full text ("1 unread discussion") to visual users on-hover, and it would automatically hook up as `aria-labelledby`.

If we did that though, I think we'd want there to always be a tab stop so that keyboard users could see it.

Reply:
I know for filter we did not want to add a tab stop, since it messes with the tab order of the menu. I like the idea of adding an optional tooltip though. There are cases, like the card-footer-link on the course cards that uses a separate tooltip; it has different text than the count badge (related to the link rather than the number).

Reply:
I chatted with Jeff, he is on board with an optional tooltip 😄 It will be opt-in since we only have a few use cases where it's needed.